public/index.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
why have you removed this ?

Reply:
It was messing up the profile page, but I will return it